katello/katello-change-hostname

```diff
@@ -170,6 +171,12 @@ def precheck
     fail_with_message("There is a problem with the server, please check 'hammer ping'")
   end
 
+  STDOUT.puts "\nChecking credentials"
+  hammer_cmd("capsule list")
+  unless $?.exitstatus == 0
```

Comment:
can we use `$?.success?` syntax here? I use it in other places in the script.

Reply:
Yes we can!